Example/app/jsx/screens/SingleExample.js

```diff
@@ -39,7 +39,7 @@ export default () => {
     ],
     "advertising": {
       "client": "googima"
-    }
+    }    
```

Comment:
Remove added whitespace
```suggestion
    }
```